src/Components/TicTacToe/Board.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+import Square from './Square';
+
+const Status = styled.div`
+    margin-bottom: 10px;
+`
+
+const BoardRow = styled.div`
+    &:after {
+        clear: both;
+        content: '';
+    }
+`
+
+class Board extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            squares: Array(9).fill(null),
+            xIsNext: true,
+        };
+    }
+
```

Comment:
If all that happens in the constructor method is the initialisation of state, it is redundant. It will work and it wont break anything but you can simply initialise the state as follows: 
```
state = {
    squares: Array(9).fill(null),
    xIsNext: true
}
```

There's no need to bother with the constructor method. Some of the older react tutorials and resources will however still show the state being initialised in the constructor method.